packages/node/src/vendor/merkle-mountain-range/fileBasedDb.ts

```diff
@@ -0,0 +1,162 @@
+// Copyright 2020-2021 OnFinality Limited authors & contributors
```

Comment:
we shouldn't claim copyright for this file I believe.
And why you adding the whole merkle-mountain-range project? we only need to make some changes in FileBasedDb I think.